src/manage_nsfs/nc_master_key_manager.js

```diff
@@ -79,6 +79,8 @@ class NCMasterKeysManager {
      */
     _set_keys(master_keys) {
         if (!master_keys.active_master_key || !master_keys.master_keys_by_id) throw new RpcError('INVALID_MASTER_KEYS_FILE', 'Invalid master_keys.json file');
+        dbg.log0('_set_keys: master_keys', util.inspect(master_keys));
```

Comment:
I don't think we want to log the actual keys, right? 
or is it wrapped in SensitiveString?
Even if so I would still avoid it to prevent mistakes, and only selectively print information about the master keys such as their ids / timestamps etc.

Reply:
@guymguym ofc not, all these prints are temporary debugging prints for the RPM I built for scale team, removing it
(see PR description - temporary prints - will remove before merging)

Reply:
@guymguym thanks to your comment I found a place where we printed the active_master_key so I removed it!

Reply:
BTW, I decided to keep some of the prints eventually